ui/src/containers/NotFound.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {MDBContainer, MDBCol, MDBRow} from 'mdbreact';
+import {Navbar} from '../components/advanced/Navbar';
+import {Footer} from '../components/advanced/Footer';
+
+export const NotFound = () => (
+  <React.Fragment>
+    <Navbar />
+    <MDBContainer>
+      <MDBRow>
+        <MDBCol sm="12" className="my-4">
+          <h1>404</h1>
```

Comment:
[IMO]
ちょっとシュールだったから、以下とかの方が良いかも
<img width="338" alt="スクリーンショット 2020-01-05 16 55 48" src="https://user-images.githubusercontent.com/28621480/71776901-54c92380-2fdc-11ea-9307-860d5ec8550c.png">

```suggestion
          <h1>ページが見つかりません</h1>
```